mesonbuild/build.py

```diff
@@ -17,6 +17,7 @@
 import copy
 import hashlib
 import itertools, pathlib
+from mesonbuild.interpreterbase.baseobjects import MesonInterpreterObject
```

Comment:
I know that vscode did this automatically for you, but we usually use relative imports rather than absolute ones. This is also in the wrong section, it should be with the other meson imports not the system ones.

Reply:
Good catch, I missed that. I probably should turn the auto-import off since it never produces good results.